### ISSUE TITLE
working with teams/ organisation. understanding problem

### ISSUE THREAD

**Issue by flobee** [go-gitea/gitea#4197]
- Gitea version (or commit ref): 1.4.2
- Git version: 2.11.0
- Operating system: debian 9 (stretch)
- Database (use `[x]`):
  - [ X] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ X] Yes 
https://try.gitea.io/company

## Description
i worked it out now to be a very little issus about the wording: "Transfer Ownership" when writing this request. But i also request for assistant: what do i need then or a how to...:
Maybe there is any doc to understand:

i create an organisation which has:
 - owner (1,2..
 - teams (eg for repo write access)
 - how to deny view access for all others? ....

the repos of the company should be private / not listed but member should have access.

actions i'v done:
```
    3. flobee transferred repository flobee/company-repo-private to company/company-repo-private
    2. flobee pushed to master at flobee/company-repo-private
    1. flobee created repository flobee/company-repo-private 
    This ---------------------------^^^^ what i'v done, not what it shows now (**flobee**/company-repo-private vs "**company**/company-repo-private"

see screenshot at the bottom. 
```
when a member of the "company" in the team loggs in eg: "test2test2" user 
test2test2 can not see any repos (https://try.gitea.io/company) 
Just if test2test2 gets owner ship. 
In one way this is correct. if a person is the owner and want to keep stuff private. as "company" or group holder? 

How to set levels like: team a only see repos they are assigned to and team b some other stuff? Its not about deny access, its about to reduce the overview of repos :) 
sounds like i need gitolite as middleware? does that work?

Repo: "**Transfer Ownership**" is in this case the kind of "bug". I'm still an owner of the repo but its not in my user account anymore. Thats what confuses me to end up in this report.
...

![2018-06-09_15-29-28](https://user-images.githubusercontent.com/1349520/41192168-52adf956-6bfa-11e8-9504-6d2f28fa6ec3.png)

**Comment by techknowlogick** [comment]
Closing issue. Please open a post in the [forum](https://discourse.gitea.io/) for this issue.

**Comment by flobee** [comment]
Seems to be implemented in version >1.7.4   :+1: 
Found in release7780ea8 at https://try.gitea.io
